topics/statistics/tutorials/fine_tuning_protTrans/slides.html

```diff
@@ -0,0 +1,174 @@
+---
+layout: tutorial_slides
+logo: GTN
+title: "Fine-tuning Protein Language Model"
+zenodo_link: "https://doi.org/10.5281/zenodo.15050767"
+questions:
+  - How to load large protein AI models?
+  - How to fine-tune such models on downstream tasks such as post-translational site prediction?
+objectives:
+  - Learn to load and use large protein models from HuggingFace
+  - Learn to fine-tune them on specific tasks such as predicting dephosphorylation sites
+key_points:
+- Training a very large deep learning model from scratch on a large dataset requires exertise and compute power 
+- Large models such as ProtTrans are trained using millions of protein sequences 
+- They contain significant knowledge about context in protein sequences
+- These models can be used in multiple ways for learning on a new dataset such as fine tuning, embedding extraction, ...
+- Fine-tuning using LoRA requires much less time and compute power 
+- Downstream tasks such as protein sequence classification can be performed using these resources
+requirements:
+  -
+    type: internal
+    topic_name: statistics
+    tutorials:
+      - machinelearning
+      - classification_machinelearning
+contributors:
+  - anuprulez
+---
+
+# Language Models (LM)
+
+- Powerful LMs "understand" language like humans
+- LMs are trained to understand and generate human language
+- Popular models: GPT-3, Llama2, Gemini, …
+- Trained on vast datasets with billions of parameters
+- Self-supervised learning
+- Masked language modeling
+- Next word/sentence prediction
+- Can we train such language models on protein/DNA/RNA sequences?
+- LM for life sciences - DNABert, ProtBert, **ProtT5**, CodonBert, RNA-FM, ESM/2, BioGPT, …
+- Many are available on HuggingFace
+
+---
+
+# Bidirectional Encoder Representations from Transformers (BERT)
+
+![BERT]({{site.baseurl}}/topics/statistics/tutorials/fine_tuning_protTrans/images/ft0.png)
+
+---
+
+# Language Models (LMs)
+
+![Rise of language models and their sizes]({{site.baseurl}}/topics/statistics/tutorials/fine_tuning_protTrans/images/ft1.png)
+
+---
+
+# Protein Language Model (pLM)
+
+- Models trained on large protein databases
+    - Big fantastic database (> 2.4 billions sequences), Uniprot, …
```

Comment:
```suggestion
    - Big fantastic database (BFD) (> 2.5 billions sequences), Uniprot, …
```

Reply:
I did not know the BFD :) and I checked here -> https://docs.unity.rc.umass.edu/documentation/datasets/bio/
They say 2.5 protein sequences. 
Maybe you can also ignore the  (> 2.4 billions sequences)? The size is also not listed for Uniport. To keep it consistent?